src/server/src/main/java/io/cassandrareaper/storage/MemoryStorage.java

```diff
@@ -193,6 +197,24 @@ public List<RepairRun> getRepairRunsForCluster(String clusterName, Optional<Inte
     return foundRepairRuns;
   }
 
+  @Override
+  public List<RepairRun> getRepairRunsForClusterPrioritiseRunning(String clusterName, Optional<Integer> limit) {
+    List<RepairRun> foundRepairRuns = repairRuns
+        .values()
+        .stream()
+        .filter(
+            row -> row.getClusterName().equals(clusterName.toLowerCase(Locale.ROOT))).collect(Collectors.toList()
+        );
+    Comparator<RepairRun> comparator = new Comparator<RepairRun>() {
+      @Override
+      public int compare(RepairRun o1, RepairRun o2) {
+        return o1.getRunState().compareTo(o2.getRunState());
+      }
+    };
+    Collections.sort(foundRepairRuns, comparator);
```

Comment:
This has one annoying side effect, which is that DONE runs will be prioritized over ABORTED ones, even if the ABORTED run is more recent than the DONE one.
That's going to be misleading for users which aren't going to find what they're looking for.
Would it be possible to create a `RunStateSet`  class of some sorts, which would group the states in two distinct sets? Something like `NOT_FINISHED` (`RUNNING`, `PAUSED`, `NOT_STARTED`) and `FINISHED` (the other states). And then we could use it as comparator to avoid reordering the runs within each set.
We need to retain the natural order within each list.

Reply:
> We need to retain the natural order within each list.

By "natural ordering", you mean time based ordering? I feel like we keep miscommunication on this bit 😬

Reply:
yes, time based ordering.

Reply:
Repair run ids are timeuuid, that should help if you need to sort them in descending order.

Reply:
All done, tests passing.

Reply:
One thing about sorting on timeuuids: it looks like right now repairs in the UI are sorted by start time, not creation time (which I think the timeuuid corresponds to). Do you want to update this behaviour to sort of creation time?